components/dashboard/src/components/mobile-menu.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import Button from "@material-ui/core/Button";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import { Branding } from "@gitpod/gitpod-protocol";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+
+const createAnchor = (name: string, url: string, handleClose: any) => {
+  if (url.startsWith("/")) {
+    return (
+      <MenuItem
+        onClick={handleClose}
+        component="a"
+        href={url}
+        key={"a-" + name}
+      >
+        {name}
+      </MenuItem>
+    );
+  } else {
+    return (
+      <MenuItem
+        onClick={handleClose}
+        component="a"
+        href={url}
+        key={"a-" + name}
+        target="_blank"
+        rel="noopener"
+      >
+        {name}
+      </MenuItem>
+    );
+  }
+};
+
+export default class MobileMenu extends React.Component<{ items: any }> {
+  state = {
+    anchorEl: null,
+  };
+
+  handleClick = (e: any) => {
+    this.setState({ anchorEl: e.currentTarget });
+  };
+
+  handleClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  render() {
+    const { items } = this.props;
+    return (
+      <div className="mobile-menu">
+        <Button
+          aria-controls="mobile-menu"
+          style={{ marginRight: "1rem" }}
+          aria-haspopup="true"
+          onClick={this.handleClick}
+        >
+          Navigation Menu <ExpandMoreIcon />
```

Comment:
I'm not sure that we want a second dropdown menu for mobile. Have you tried moving the horizontal menu elements into the already-existing dropdown on mobile?

E.g. like so:

```
(desktop)
------------------------------------------------------------------------------
<G>Gitpod        Workspaces  Docs  Blog  Community                        [O]v
                                                               ---------------
                                                               My Usage
                                                               My Subscription
                                                               My Teams
                                                               Access Control
                                                               Settings
                                                               Log Out
                                                               ---------------
```

```
(mobile)
-------------------------------------------
<G>Gitpod                              [O]v
                            ---------------
                            Workspaces
                            Docs
                            Blog
                            Community
                            ---------------
                            My Usage
                            My Subscription
                            My Teams
                            Access Control
                            Settings
                            Log Out
                            ---------------
```